mapie/quantile_regression.py

```diff
@@ -363,16 +369,41 @@ def fit(  # type: ignore
         alpha = self._check_alpha(self.alpha)
         self.cv = self._check_cv(cast(str, self.cv))
         X, y = indexable(X, y)
+        if X_calib is None or y_calib is None:
+            if sample_weight is not None:
+                (
+                    X_train,
+                    X_calib,
+                    y_train,
+                    y_calib,
+                    sample_weight_train,
+                    _,
+                ) = train_test_split(
```

Comment:
Be careful, `random_state`, `shuffle`, and `stratify` are not defined and should be added as additional arguments in the fit method.

Reply:
done